app/main.py

```diff
@@ -0,0 +1,16 @@
+from fastapi import FastAPI
+
+import fast_api_db.utils as utils
+
+# attempt to replace the output of read_message
+# from src.fast_api_db.main import read_message
+
+app = FastAPI()
+
+
+@app.get("/")
+def utils_function():
+    return utils.dummy_function()
```

Comment:
it does not return a payload since print returns Null. Can you make it to {'message': 'dummy_function'}?